udata/settings.py

```diff
@@ -187,6 +187,7 @@ class Defaults(object):
         'a': ['href', 'title', 'rel', 'data-tooltip'],
         'abbr': ['title'],
         'acronym': ['title'],
+        'img': []
```

Comment:
Why aren’t we using `src` and `title` here? It seems to work for the other attrs.